admin/updates/update-2-add_dataset_eval_location.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
```

Comment:
@gentlecat suggested here that we change the filenames of our update files to have the current date `yyyy-mm-dd` instead of a number, this way if two people make a change to alter the database at the same time we won't have conflicting numbers. Can you change this?

Make sure you don't have 3 newlines at the end of the file